command/helpers.go

```diff
@@ -542,9 +545,20 @@ func (j *JobGetter) Get(jpath string) (*api.JobSubmission, *api.Job, error) {
 			Strict:   j.Strict,
 		})
 
+		var varFileCat string
+		var readVarFileErr error
+		if err == nil {
```

Comment:
This is `if err == nil`; we only bother reading the var files with `ParseWithConfig` was successful. The error handling for this whole switch statement is clunky - because we want to keep track if hcl2 parsing failed, then try hcl1 parsing just so we can provide a customized error message outside the switch. When we finally drop hcl1 I imagine a lot of this just goes away.